index.dd

```diff
@@ -123,6 +123,21 @@ void main()
 }
 ----
 )
+$(EXTRA_EXAMPLE_RUNNABLE Print hex dump,
+----
+void main()
+{
+    import std.algorithm, std.stdio, std.file;
+    enum cols = 16;
+    thisExePath.File("rb").byChunk(cols).each!(chunk =>
+        writefln!"%(%02X %)%*s  %s"(
```

Comment:
Another suggestion is to use the version of `writefln` that takes a runtime since all C and C++ programmers should understand those.